src/main/java/org/reactome/release/qa/check/NewRegulationChecker.java

```diff
@@ -0,0 +1,120 @@
+    {
+        this.priorAdaptor = adaptor;
+    }
+
+    @Override
+    public QAReport executeQACheck() throws Exception
+    {
+        QAReport report = new QAReport();
+        report.setColumnHeaders("DBID","DisplayName", "SchemaClass", "Issue", "MostRecentAuthor");
+        // Find all ReactionlikeEvents that have a filled 'regulatedBy' attribute in current slice.
+        Set<GKInstance> currentRlEsWithRegulations = getRlEsWithRegulationInstances(dba);
+
+        for (GKInstance currentRlE : currentRlEsWithRegulations) {
+            GKInstance previousRlE = priorAdaptor.fetchInstance(currentRlE.getDBID());
+            // QA check
+            if (changedRegulatedByWithoutNewReviewed(currentRlE, previousRlE)){
+                report.addLine(
+                        currentRlE.getDBID().toString(),
+                        currentRlE.getDisplayName(),
+                        currentRlE.getSchemClass().getName(),
+                        "ReactionlikeEvent with new regulatedBy instance but has not yet been reviewed",
+                        QACheckerHelper.getLastModificationAuthor(currentRlE)
+                );
+            }
+        }
+        return report;
+    }
+
+    /**
+     * This method checks to see if a new regulation instance has been added to the ReactionlikeEvent by comparing between slices.
+     * If it does, it then compares the number of reviewed instances between slices. If the number of regulation instances is greater AND
+     * the number of reviewed instances is the same, it is flagged to indicate that this ReactionlikeEvent still needs to reviewed before
+     * it can be released.
+     * @param currentRlE GKInstance -- ReactionlikeEvent from current slice.
+     * @param previousRlE GKInstance -- Same ReactionlikeEvent, but from previous slice.
+     * @return boolean -- True if a new, unreviewed regulation instance has been added.
+     * @throws Exception -- DBA exceptions.
+     */
+    private boolean changedRegulatedByWithoutNewReviewed(GKInstance currentRlE, GKInstance previousRlE) throws Exception {
+
+        // If previousRlE does not exist, we ignore it since that does not pertain to this particular QA check.
+        if (previousRlE == null) {
+            return false;
+        }
+        // Compare contents of the 'regulatedBy' attribute of the current and previous versions of the RlE.
+        boolean sameRegulatedBy = isSameRegulatedByValues(
+                currentRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy),
+                previousRlE.getAttributeValuesList(ReactomeJavaConstants.regulatedBy)
+        );
+
+        int currentRlEReviewed = currentRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+        int previousRlEReviewed = previousRlE.getAttributeValuesList(ReactomeJavaConstants.reviewed).size();
+
+        // The actual QA check
+        return !sameRegulatedBy && currentRlEReviewed == previousRlEReviewed;
```

Comment:
The count-based comparison is always error-prone. We have to assume the curator doesn't make any change to the reviewed slot. Probably it is fine for this specific slot.